operator/apis/platform/v1alpha1/central_types.go

```diff
@@ -213,11 +214,8 @@ type CentralDBSpec struct {
 type CentralDBEnabled string
 
 const (
-	// CentralDBEnabledDefault configures the central to use rocksdb
-	// TODO(ROX-13557): in 3.74.0 this should say that Default enables PostgreSQL.
+	// CentralDBEnabledDefault configures the central to use PostgreSQL database.
 	CentralDBEnabledDefault CentralDBEnabled = "Default"
-	// CentralDBEnabledTrue configures the central to use a PostgreSQL database (Technology Preview)
-	CentralDBEnabledTrue CentralDBEnabled = "Enabled"
 )
 
 // CentralDBEnabledPtr return a pointer for the given CentralDBEnabled value
```

Comment:
I believe this function will no longer needed once the other suggestions are applied, and can be removed?